rosa/rosa-wrapper.py

```diff
@@ -106,6 +106,24 @@ def _install_addons(rosa_cmnd,cluster_id,addons):
             logging.error(addon_stderr.strip().decode("utf-8"))
         # TODO: control addon is installed with: rosa list addons -c <<cluster_name>>
 
+def _expiration(rosa_cmnd,cluster_id,expiration):
+    logging.info('Extending cluster expiration for %d minutes on %s' % (expiration,cluster_id))
+    # rosa edit cluster -c 1iaaehmdd23lhifqk4fsjghrci82nt51 --expiration=3600m
+    expiration_cmd = [rosa_cmnd, "edit", "cluster",
+                      "--cluster", cluster_id,
+                      "--expiration", str(expiration)+'m']
+    logging.debug(expiration_cmd)
+    expiration_process = subprocess.Popen(expiration_cmd, stdout=subprocess.PIPE, stderr=subprocess.PIPE)
+    expiration_stdout,expiration_stderr = expiration_process.communicate()
+    if expiration_process.returncode != 0:
+        logging.error('Unable to extend expiration on %s' % cluster_id)
+        logging.error(expiration_stdout.strip().decode("utf-8"))
+        logging.error(expiration_stderr.strip().decode("utf-8"))
+        return 1
```

Comment:
(nitpick) It would be more "Pythonic" to throw an Exception as opposed to returning a number. e.g.

```
raise SomeException("Unable to extend expiration on %s" % cluster_id)
```

Reply:
There are no exceptions on the code.

We could move to them, but we would have to do it on all the code. Adding an issue for a future refactor